src/tooltip/tooltip.js

```diff
@@ -302,6 +299,18 @@ angular.module( 'ui.bootstrap.tooltip', [ 'ui.bootstrap.position', 'ui.bootstrap
 
             attrs.$observe( prefix+'Title', function ( val ) {
               ttScope.title = val;
+              if (prefix === 'popover') {
```

Comment:
It's probably better to remove the check since any user of the `*Title` attribute would expect a re-render if the attribute's value changes. Users can make use of the `title` variable in the ttScope in their custom tooltips.